tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -61,8 +63,16 @@ func Cmd(logOpts *log.Options) *cobra.Command {
 		Use:          "bug-report",
 		Short:        "Cluster information and log capture support tool.",
 		SilenceUsage: true,
-		Long: "This command selectively captures cluster information and logs into an archive to help " +
-			"diagnose problems. It optionally uploads the archive to a GCS bucket.",
+		Long: "bug-report selectively captures cluster information and logs into an archive to help " +
+			"diagnose problems and optionally uploads the archive to a GCS bucket. \n" +
+			"Proxy logs can be filtered using:\n" +
+			"  --include|--exclude namespace1,.../deployment1,.../pod1,.../container1,.../label1=value1,.../annotation1=value1,...\n" +
+			"Exclude is applied after include." +
+			"All components are optional and can be omitted, which selects all.\n" +
+			"All names except label and annotation keys support '*' glob matching pattern.\n" +
```

Comment:
Do we log warning message when user try to use '*' for label and annotation keys or just silently process

Reply:
we didn't, but we do now :-)